contracts/SimpleArtistContract.sol

```diff
@@ -0,0 +1,177 @@
+pragma solidity ^0.4.21;
+
+import "./DART.sol";
+
+import "./Ownable.sol";
+
+/**
+* @title SimpleArtistContract
+*/
+contract SimpleArtistContract is Ownable {
+  using SafeMath for uint256;
+
+  event Purchase(address indexed _funder, uint256 indexed _tokenId, bytes32 _blockhash, uint256 _block);
+  event Purchased(address indexed _funder, uint256 indexed _tokenId, uint256 _blocksPurchased);
+
+  modifier onlyValidAmounts() {
+    require(msg.value >= 0);
+
+    // Min price
+    require(msg.value >= pricePerBlock);
+
+    // max price
+    require(msg.value <= (pricePerBlock * maxBlockPurchaseInOneGo));
+    _;
+  }
+
+  DART public token;
+
+  uint256 public pricePerBlock = 0.01 ether;
```

Comment:
Setter required? - could take default from DART token and then allow config per contract. Same from maxBlocks etc